internal/command/meta.go

```diff
@@ -58,30 +58,53 @@ func (c *Meta) resolveStatus(err error) Status {
 	return Success
 }
 
-func (c *Meta) addOutput(key string, value string) {
-	// check if context in not github/gitlab
-	if c.Env.Context != nil {
-		c.Env.Context.AddOutput(key, value)
-	} else {
-		// cli use
-		c.cliOut[key] = value
-	}
+// adds new output value to map as &OutputMessage{}
+func (c *Meta) addOutput(name string, value string) {
+	c.messageOutput[name] = newOutputMessage(name, value)
+}
+
+type outputOpts struct {
+	// indicates if value should be displayed to stdout
+	stdOut bool
+	// indicates if value contains a multiline value as some platforms: gitlab do not support multiline values in `.env`
+	multiLine bool
+}
+
+// adds new output value with options &outputOpts{}
+func (c *Meta) addOutputWithOpts(name string, value interface{}, opts *outputOpts) {
+	msg := newOutputMessage(name, value)
```

Comment:
Maybe we don't need to set `stdOut` and `multiLine` from outside.
How about we pass `outputOpts` as an argument into `newOutputMessage`. The revised definition would be:
```go
func newOutputMessage(name, value, *outputOpts)
```

Reply:
Yeah great suggestion! 👍